src/components/MarcusFEMentor2/styles/Footers.styled.ts

```diff
@@ -0,0 +1,131 @@
+import styled from 'styled-components'
+
+import {
+  darkGreen,
+  lightCoral,
+  sacramentoStateGreen,
+} from '../../../pages/FrontendMentor/MarcusFEMentor2/styles/Colours.styled'
+import { PageDiv, SocialMediaIconHover } from '../../../pages/FrontendMentor/MarcusFEMentor2/styles/DivStyles.styled'
+import { h2Styled, body2 } from '../../../pages/FrontendMentor/MarcusFEMentor2/styles/TextStyles.styled'
+
+// width 1440px (rel), height 745px (abs)
```

Comment:
intentional? if it is, maybe add some explanation, otherwise should be removed or